.travis.yml

```diff
@@ -16,8 +16,10 @@ env:
   - IMAGEMAGICK_VERSION=6.7.7-10 CONFIGURE_OPTIONS=--enable-hdri
   - IMAGEMAGICK_VERSION=6.8.9-10
   - IMAGEMAGICK_VERSION=6.8.9-10 CONFIGURE_OPTIONS=--enable-hdri
-  - IMAGEMAGICK_VERSION=6.9.10-35
-  - IMAGEMAGICK_VERSION=6.9.10-35 CONFIGURE_OPTIONS=--enable-hdri
+  - IMAGEMAGICK_VERSION=6.9.9-51
```

Comment:
We really don't need to add an extra ImageMagick version.

Reply:
Done